src/DynamicFormBuilder.jsx

```diff
@@ -261,6 +261,13 @@ class DynamicFormBuilder extends React.Component {
         return !invalid;
     }
 
+    getInputValidationError(inputName) {
+        const validationError = this.state.validation_errors[inputName];
+        const propError = this.props.formErrors[inputName];
```

Comment:
Our internal JS standards say that objects (primarily `state` and `props`) should be deconstructed, for example:
```js
const { formErrors } = this.props;
```